src/scripts.js

```diff
@@ -333,16 +310,20 @@ function populateCards(arr){
   allRecipeGrid.innerHTML = ""
   const recipeCard = arr.reduce((acc, recipe) => {
 
-    let buttonClasses = "favorite-star"
+    let buttonClassesFaves = "favorite-star"
     if (currentUser.favoriteRecipes.includes(recipe)){
-      buttonClasses = "favorite-star is-favorite heart-button"
+      buttonClassesFaves = "favorite-star is-favorite"
+    }
+    let buttonClassesToCook = "recipesToCook"
+    if (currentUser.recipesToCook.includes(recipe)){
+      buttonClassesToCook = "recipesToCook is-saved"
     }
     allRecipeGrid.innerHTML +=
       `<article class="mini-recipe" id="${recipe.id}">
        <img src= "${recipe.image}" alt= "${recipe.name}">
        <p>${recipe.name}</p>
-       <button type="favoriteStar" name="favoriteStar" class="favorite-star${buttonClasses}" id="fave-${recipe.id}">♡</button>
-       <button type="recipesToCook" name="recipesToCook" class="recipesToCook" id="${recipe.id}">🗓 Cook this week!</button>
+       <button type="favoriteStar" name="favoriteStar" class="${buttonClassesFaves}" id="fave-${recipe.id}">♡</button>
+       <button type="recipesToCook" name="recipesToCook" class="${buttonClassesToCook}" id="${recipe.id}">🗓 Cook this week!</button>
```

Comment:
thanks for drying this function up a bit. I think we still have some work to do (maybe create some more helper functions, but we're on the right track)